content/engine/security/rootless.md

```diff
@@ -581,23 +581,47 @@ $ systemctl --user restart docker
 
 **`docker run -p` does not propagate source IP addresses**
 
-This is because Docker with rootless mode uses RootlessKit's builtin port driver by default.
+This is because Docker in rootless mode uses RootlessKit's `builtin` port
+driver by default, which doesn't support source IP propagation. To enable
+source IP propagation, you can:
 
-The source IP addresses can be propagated by creating `~/.config/systemd/user/docker.service.d/override.conf` with the following content:
+- Use the `slirp4netns` RootlessKit port driver
+- Use the `pasta` RootlessKit network driver, with the `implicit` port driver
 
-```systemd
-[Service]
-Environment="DOCKERD_ROOTLESS_ROOTLESSKIT_PORT_DRIVER=slirp4netns"
-```
+The `pasta` network driver is experimental, but provides improved throughput
+performance compared the `slirp4netns` port driver.
```

Comment:
Maybe:
```suggestion
performance compared to the `slirp4netns` port driver.
```

Reply:
It should be also noted that the pasta driver needs RootlessKit v2 (will be included in Docker v25)